### PR TITLE
iz GAWB-1293, GAWB-1294, GAWB-1295

### PR DESCRIPTION
**Pull request by zarsky-broad**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/org/broadinstitute/firecloud_ui/page/billing/create_project.cljs [hunk 1]
```diff
@@ -38,88 +33,86 @@
...
-         (let [[name & fails] (input/get-and-validate refs "name-field")]
-           (swap! state assoc :validation-errors fails)
-           (when-not fails
+       (when-not account
+         (swap! state assoc :account-errors ["Please select a billing account"]))
+       (let [[name & fails] (input/get-and-validate refs "name-field")]
+         (swap! state assoc :validation-errors fails)
+         (when-not (and fails (not account))
+           (do
+             (js* "debugger;")
```
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/page/billing/create_project.cljs]: Whoops?

**Review by dmohs — CHANGES_REQUESTED** [review]
Since I just touched this, lets have you rebase before I review.

File: src/cljs/org/broadinstitute/firecloud_ui/page/billing/billing_management.cljs [hunk 2]
```diff
@@ -84,11 +84,26 @@
...
-                                      {:on-success #(react/call :reload this)}]))}])
+          [comps/Button
+           {:text "Create New Billing Project"
+            :onClick (fn [_]
+                       (if (-> @utils/google-auth2-instance (aget "currentUser") (js-invoke "get")
+                               (js-invoke "hasGrantedScopes" "https://www.googleapis.com/auth/cloud-billing"))
+                         (modal/push-modal
+                          [CreateBillingProjectDialog
+                           {:on-success #(react/call :reload this)}])
+                         (fn [{:keys [this]} scopes-needed]
```
Comment (dmohs) [inline src/cljs/org/broadinstitute/firecloud_ui/page/billing/billing_management.cljs]: This branch of the if statement does not work.

**Review by dmohs — CHANGES_REQUESTED** [review]

File: src/cljs/org/broadinstitute/firecloud_ui/page/billing/billing_management.cljs (hunk 2 above)
Comment (dmohs) [reply src/cljs/org/broadinstitute/firecloud_ui/page/billing/billing_management.cljs]: Call this in your console to test:
```javascript
gapi.auth2.getAuthInstance().currentUser.get().disconnect()
```

Comment (zarsky-broad) [reply src/cljs/org/broadinstitute/firecloud_ui/page/billing/billing_management.cljs]: I think this is fixed.

**Review by dmohs — APPROVED** [review]
Looks good to me, but I think this requires the following change in Rawls: https://github.com/broadinstitute/rawls/pull/601
@davidangb I think this would address the issue you pointed out to me in a previous PR.

**Pull request merged**